projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts

```diff
@@ -10,47 +10,96 @@ import {CONTEXT_TOKEN, LayerCollectionListComponent} from '../layer-collection-l
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LayerCollectionNavigationComponent {
-    collections: Array<LayerCollectionItemDict | undefined> = [undefined];
+    collections: Array<LayerCollectionItemDict> = [];
+    allTrails: Array<Array<LayerCollectionItemDict>> = [];
+    displayedTrail: Array<LayerCollectionItemDict> = [];
 
-    selectedCollection = 0;
+    selectedCollection = -1;
 
     selectedPortal!: Portal<any>;
 
+    @ViewChild('scrollElement', {read: ElementRef}) public scrollElement!: ElementRef<any>;
+
     constructor() {
         this.setPortal(undefined);
     }
 
+    scrollToRight(): void {
+        setTimeout(() => {
+            // wait until breadcrumbs are re-rendered before scrolling
+            this.scrollElement.nativeElement.scrollLeft += 500;
```

Comment:
will scrolling 500 always work? Maybe you could calculate the width of the div and set it to the correct value instead.

Reply:
I scrolled to the right by the max possible value (maybe a bit of a sledgehammer approach, but it saves having to calculate widths) 4a8a0b13e38c962311cf7b0d4654be79406cd047